composer.json

```diff
@@ -18,7 +18,7 @@
     "require-dev": {
         "symfony/config": "2.*",
         "phpunit/phpunit": "3.7.*",
```

Comment:
Okay, I'll change that. We should then also use this version to run tests on Travis instead of their version, shouldn't we?